### DISCUSSION TITLE
Digital Ocean Droplet - 1GB RAM not enough?

### DISCUSSION THREAD

**Discussion by farmserve** [Budibase/budibase#6634] [Help]
I have a droplet with an empty Budibase instance (no apps) running on a basic shared CPU with 1GB RAM and 1 CPU, however, I am experiencing frequent timeouts or issues logging into my instance. I also notice that the RAM is constantly at 100%. Is anyone else experiencing this on a 1GB instance? Is the only solution to upgrade to a 2GB instance? If this is the case, shouldn't the minimum viable option be a 2GB instance?

**Comment by chadgriff** [comment]
@farmserve I too started with a 1GB RAM sized Droplet and experienced issues. I sized up to a 2GB Droplet and it has been running fine ever since (it is not the fastest but fine for now as I don't have many users on BB just yet). I plan to scale up the Droplet size as needed as my user base grows.
